UNITTESTS/stubs/storage/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(mbed-stubs-storage)
+
+target_sources(mbed-stubs-storage
+    PRIVATE
+        BufferedBlockDevice_stub.cpp
+        ChainingBlockDevice_stub.cpp
+        EmulatedSD.cpp
+        ExhaustibleBlockDevice_stub.cpp
+        FileHandle_stub.cpp
+        FlashSimBlockDevice_stub.cpp
+        HeapBlockDevice_stub.cpp
+        #MBRBlockDevice_stub.cpp
```

Comment:
appears broken and unused - this needs to be checked if any tests actually use it